actions.rb

```diff
@@ -0,0 +1,48 @@
+require_relative 'app'
+
+# def create_person_option
+#   puts 'Do you want to create a Student (1) or a Teacher (2)? [Input the number]:'
+#   gets.chomp.to_i
+# end
```

Comment:
- [ Required ] I think it would be nice if you remove any commented pieces of code from your code base, commented code gives your code base an unprofessional look and is also considered a sign of code that is not clean.